lib/consul/controller.rb

```diff
@@ -43,8 +43,10 @@ def require_power_check(options = {})
       def skip_power_check(options = {})
         if Rails.version.to_i < 4
           skip_before_filter :unchecked_power, options
-        else
+        elsif Rails.version.to_i < 5
           skip_before_action :unchecked_power, options
+        else
+          skip_before_action :unchecked_power, { :raise => false }.merge!(options)
```

Comment:
FYI, it's documented in the Rails docs for `skip_callback` (which is called by `skip_before_action`): http://api.rubyonrails.org/classes/ActiveSupport/Callbacks/ClassMethods.html#method-i-skip_callback